SDWebImage/SDImageAPNGCoder.m

```diff
@@ -83,10 +83,7 @@ - (UIImage *)decodedImageWithData:(NSData *)data options:(nullable SDImageCoderO
     CGFloat scale = 1;
     NSNumber *scaleFactor = options[SDImageCoderDecodeScaleFactor];
     if (scaleFactor != nil) {
```

Comment:
@kinarob I think you can even remove all these nil check, just one line `scale = MAX([scaleFactor doubleValue], 1);`. 😄

Reply:
Good point！